packages/lit-auth-client/src/lib/lit-auth-client.ts

```diff
@@ -1,395 +0,0 @@
-        );
-    }
-
-    this.providers.set(type, provider);
-    return provider;
-  }
-
-  /**
-   * Returns an initialized provider by type
-   *
-   * @param {ProviderType} type - Type of provider to get
-   *
-   * @returns {BaseProvider | undefined} - Provider if found, undefined otherwise
-   */
-  getProvider(type: ProviderType): BaseProvider | undefined {
-    return this.providers.get(type);
-  }
-
-  /**
-   * Retrieves the authentication ID based on the provided authentication method.
-   *
-   * @param {AuthMethod} authMethod - The authentication method
-   * @returns {Promise<string>} - The authentication ID
-   */
-  public static async getAuthIdByAuthMethod(
-    authMethod: AuthMethod
-  ): Promise<string> {
-    let authId;
-
-    switch (authMethod.authMethodType) {
-      case AuthMethodType.EthWallet:
-        authId = await EthWalletProvider.authMethodId(authMethod);
-        break;
-      case AuthMethodType.Discord:
-        authId = await DiscordProvider.authMethodId(authMethod);
-        break;
-      case AuthMethodType.WebAuthn:
-        authId = await WebAuthnProvider.authMethodId(authMethod);
-        break;
-      case AuthMethodType.GoogleJwt:
-        authId = await GoogleProvider.authMethodId(authMethod);
-        break;
-      case AuthMethodType.StytchOtp:
-        authId = await StytchOtpProvider.authMethodId(authMethod);
-        break;
-      case AuthMethodType.StytchEmailFactorOtp:
-      case AuthMethodType.StytchSmsFactorOtp:
-      case AuthMethodType.StytchTotpFactorOtp:
-      case AuthMethodType.StytchWhatsAppFactorOtp:
-        authId = await StytchAuthFactorOtpProvider.authMethodId(authMethod);
-        break;
-      default:
-        log(`unsupported AuthMethodType: ${authMethod.authMethodType}`);
-        throw new Error(
-          `Unsupported auth method type: ${authMethod.authMethodType}`
-        );
-    }
-
-    return authId;
-  }
-
-  /**
-   * Mints a new pkp with all AuthMethods provided. Allows for permissions and flags to be set seperately.
-   * If no permissions are provided then each auth method will be assigned `1` for sign anything
-   * If no flags are provided then `sendPkpToitself` will be false, and `addPkpEthAddressAsPermittedAddress` will be true
-   * It is then up to the implementor to transfer the pkp nft to the pkp address.
-   * **note** When adding permissions, each permission should be added in the same order the auth methods are ordered
-   * @throws {Error} - Throws an error if no AuthMethods are given
-   * @param {AuthMethod[]} - AuthMethods authentication methods to be added to the pkp
-   * @param {{ pkpPermissionScopes?: number[][]; sendPkpToitself?: boolean; addPkpEthAddressAsPermittedAddress?: boolean;}}
-   * @returns {Promise<{pkpTokenId?: string; pkpEthAddress?: string; pkpPublicKey?: string}>} pkp information
-   */
-  public async mintPKPWithAuthMethods(
```

Comment:
I think we need to port this method to `relay.ts` since we will need an upgrade path for users who relied on this method.

Reply:
Good call, restored